srcpkgs/ncspot/template

```diff
@@ -32,3 +32,8 @@ post_install() {
 	vcompletion misc/_ncspot zsh
 	vcompletion misc/ncspot.fish fish
 }
+
+# FIXME: Workaround. Should be removed when https://github.com/aws/aws-lc-rs/issues/632 is resolved.
+if [ "$XBPS_TARGET_MACHINE" = armv7l ]; then
+	export AWS_LC_SYS_CFLAGS="-Wno-stringop-overflow"
+fi
```

Comment:
this if block should go before functions

Reply:
This if is not enough, because armv6l won't compile. This is why I proposed the case like the template of spotify-player

Reply:
that too